### PR TITLE
docs(configs): adds readme for configs package

### PR DESCRIPTION
**Pull request by mathewmeconry**
## Description

adds readme file for the osx-commons-configs package

Task ID: [OS-929](https://aragonassociation.atlassian.net/browse/OS-929)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.

[OS-929]: https://aragonassociation.atlassian.net/browse/OS-929?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Rekard0 — APPROVED** [review]
LGTM!

File: configs/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Aragon OSx Configs Commons
+
+The `@aragon/osx-commons-configs` provides configurations and helpers for said
+configurations necessary for the development of plugins and the OSx protocol.
+This includes:
+
+- Networks
+  - RPC url
+  - if it is a testnet or not
+  - chainID
```
Comment (Michael-A-Heuer) [inline configs/README.md:10]:
```suggestion
  - chain ID
```

File: configs/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  - alias for ethers and Alchemy subgraph
+- Deployments of the OSx protocol grouped by network
+  - Version
+  - Contract name
+  - Contract address
+  - Deployment transaction
+  - Block number when the contract was deployed (This can differ from when the
```
Comment (Michael-A-Heuer) [inline configs/README.md:17]:
```suggestion
  - Block number when the contract was deployed (which can differ from when the
```

**Review by Michael-A-Heuer — COMMENTED** [review]
Minor improvements. Otherwise, good to go.

File: configs/README.md (hunk 2 above)
Comment (mathewmeconry) [reply configs/README.md:17]: done

File: configs/README.md (hunk 1 above)
Comment (mathewmeconry) [reply configs/README.md:10]: done

**Review by Michael-A-Heuer — APPROVED** [review]

**Review by Rekard0 — APPROVED** [review]

**Pull request merged**